vdx_vtep/actions/vdx_ssh/ssh.py

```diff
@@ -0,0 +1,165 @@
+# Copyright 2017 Great Software Laboratory Pvt. Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import paramiko
```

Comment:
Thanks for suggestion @LindsayHill .
I have updated the code as per your suggestions..
 
Now we are using netmiko instead of paramiko.